sparkly/testing.py

```diff
@@ -120,28 +147,132 @@ def tearDown(self):
         for fixture in self.fixtures:
             fixture.teardown_data()
 
-    def assertDataFrameEqual(self, actual_df, expected_data, fields=None, ordered=False):
+    def assertDataFrameEqual(self,
+                             actual_df,
+                             expected_data,
+                             fields=None,
+                             ordered=False,
+                             places=None):
         """Ensure that DataFrame has the right data inside.
 
         Args:
             actual_df (pyspark.sql.DataFrame|list[pyspark.sql.Row]): Dataframe to test data in.
             expected_data (list[dict]): Expected dataframe rows defined as dicts.
             fields (list[str]): Compare only certain fields.
             ordered (bool): Does order of rows matter?
+            places (int|None): Compare floats up to given decimal places.
```

Comment:
Hm when comparing floats you do need two switches: relative tolerance (deviation is not more than 0.2) and absolute tolerance (deviation is not more than 10% of float's magnitude). I think NumPy's API is pretty standard.

With our previous API, we used `places` and `significant` as proxies to achieve the same effect. Where you could round up to X decimal places or X significant digits. It was more confusing that the current API, so I got rid of it completely - we don't have `places` anymore (although python's `assertAlmostEqual` also uses it). 

As said, yes, now you can check deviation is not more than 0.2 by just setting `atol=0.2`.